tests/contrib/elasticsearch/test.py

```diff
@@ -62,6 +62,7 @@ def test_elasticsearch(self):
         eq_(span.span_type, "elasticsearch")
         eq_(span.error, 0)
         eq_(span.get_tag(metadata.METHOD), "PUT")
+        eq_(span.get_tag(metadata.STATUS), u'200')
```

Comment:
let's try to recreate in tests a few scenarios where elasticsearch returns a non-200 error code, and make sure we capture it

e.g.
1. `es.get(index="non_existent_index", id="42) `  should return a `404`
2. `es.indices.create(index="index_that_already_exists") ` should return a `400`

... and if you can think of any others